staging/src/k8s.io/apiserver/pkg/util/flowcontrol/apf_controller.go

```diff
@@ -67,6 +67,22 @@ const timeFmt = "2006-01-02T15:04:05.999"
 // undesired becomes completely unused, all the config objects are
 // read and processed as a whole.
 
+const (
+	// Borrowing among priority levels will be accomplished by periodically
+	// adjusting the current concurrency limits (CurrentCLs);
+	// borrowingAdjustmentPeriod is that period.
+	borrowingAdjustmentPeriod = 10 * time.Second //nolint:golint,unused
+
+	// The input to the borrowing is smoothed seat demand figures.
+	// Every adjustment period, each priority level's new smoothed demand is set to
+	// seatDemandSmoothingCoefficient times the previous smoothed demand
+	// plus (1-seatDemandSmoothingCoefficient) times the recent envelope
```

Comment:
Can you mention that this is a low-pass filter? I think that will make it easier for casual readers to understand / search for more information.

Reply:
By `max` you mean the `envelope` in the text? Maybe you can add a sentence expanding what goes into the envelope computation or leaving a breadcrumb.

(It's still a low-pass filter even if it's chained after some other kind of filter :) )

Reply:
Oh dang!  Typo here, I left out the `max` part.  No wonder you're confused.  The KEP has the right equation, in https://github.com/kubernetes/enhancements/tree/master/keps/sig-api-machinery/1040-priority-and-fairness#dispatching .  It is

```
SmoothSeatDemand(i) := max( EnvelopeSeatDemand(i), A*SmoothSeatDemand(i) + (1-A)*EnvelopeSeatDemand(i) )
```

Reply:
Ah, that makes more sense, I would personally describe this as something like "seatDemandSmoothingCoefficient tells how quickly smoothed demand decays when it is greater than the current envelope" and perhaps leave the formula in the code and KEP?